instagram-app/src/components/CommentSection/Comment.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+
+const Comment = props =>{
+    return (
+        <div className = 'comment'>
+            <span className ='user'>{props.comment.username}</span>
+            <span className = 'comment'>{props.comment.text}</span>{' '}            
+        </div>
```

Comment:
I'd highly recommend adding some space between the two `span` elements for the username and text (perhaps even style the username text differently).